CHANGELOG.md

```diff
@@ -11,18 +11,23 @@ _This release is scheduled to be released on 2020-10-01._
 
 ### Added
 
+- Added a dependency on 'node-ical' and 'moment-timezone' libraries
```

Comment:
are there any direct benefits froms witching to node-ical?

Reply:
From what I can see, 'ical' is mainly targeted for pure Javascript environments whereas 'node-ical' will provide more flexibility/functionality in Node environments. This was evident when I tried fixing an incorrect issue parsing start-time timezone using 'ical', but couldn't do it without having to copy all of the timezone database data.